package NYT.NYTree.NProto;

option go_package = "a.yandex-team.ru/yt/go/proto/core/ytree";

////////////////////////////////////////////////////////////////////////////////

message TReadRequestComplexityLimits
{
    optional int64 node_count = 1;
    optional int64 result_size = 2;
}

////////////////////////////////////////////////////////////////////////////////
